app/presenters/order_detail_presenter.rb

```diff
@@ -0,0 +1,64 @@
+class OrderDetailPresenter < SimpleDelegator
+
+  include ActionView::Helpers::NumberHelper
+  include DateHelper
+
+  delegate :admin_editable?, to: :reservation, prefix: true
+  delegate :name, to: :order_status, prefix: true
```

Comment:
This seems to be doing the same thing that #405 is doing. I think it would be nice to converge on a single thing. The "presenter"/"decorator" distinction is always a little weird. Theoretically you could do a couple layers of wrapping, but that seems overkill.

Looking at OrderStatus, we might not even need this because it already defines `to_s` as `name`. So in HAML, `= order_detail.order_status` should render out to the status's name already.